### PR TITLE
Added documentation for Data Enrichment

### PR DESCRIPTION
**Pull request by mptap**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,6 +49,69 @@ environment-based approach because it keeps sensitive information out of the cod
 
 Once your environment is properly configured, the `Keen` object is ready to go immediately.
 
+### Data Enrichment
+
+A data enrichment is a powerful add-on to enrich the data you're already streaming to Keen IO by pre-processing the data and adding helpful data properties. To activate add-ons, you simply add some new properties within the "keen" namespace in your events. Detailed documentation for the configuration of our add-ons is available `here <https://keen.io/docs/api/ruby#data-enrichment>`.
+
+Here is an example of using the `URL parser <https://keen.io/docs/streams/data-enrichment-overview/#addon-url-parser>`:
```
Comment (tbarn) [inline README.md]: You will want to make sure to use this format for links `[link text](link)`, without the ` marks.

File: README.md [hunk 2]
```diff
@@ -49,6 +49,69 @@ environment-based approach because it keeps sensitive information out of the cod
...
+
+```ruby
+    keen.add_event("requests", {
```
Comment (tbarn) [inline README.md]: This is the incorrect syntax for sending an event with the Ruby gem. You need to use `Keen.publish(...)`. The readme has more docs on this.

**Comment by mptap** [comment]
@tbarn: Updated, thanks!

**Comment by tbarn** [comment]
Thanks for the contribution @mptap! 🎉 Could you fill out this form? https://goo.gl/forms/HgAERcDj3fkQL3vE3

We are low on some shirts, but we might be able to send it something else instead.

**Comment by mptap** [comment]
Done! Thanks @tbarn!

**Pull request merged**